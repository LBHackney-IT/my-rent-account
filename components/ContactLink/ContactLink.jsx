import React from "react";
import PropTypes from "prop-types";

const ContactLink = ({ contactEmail, contactNumber }) => {
  return (
    <div>
      <p className="govuk-body">
        If you have an account query, please email us at{" "}
        <a href={contactEmail}>{contactEmail}</a>, or call us on {contactNumber}
      </p>
    </div>
  );
};

ContactLink.propTypes = {
  contactEmail: PropTypes.string.isRequired,
  contactNumber: PropTypes.string.isRequired,
};

export default ContactLink;
