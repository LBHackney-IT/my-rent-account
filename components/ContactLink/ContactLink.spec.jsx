import React from "react";
import { render } from "@testing-library/react";

import ContactLink from "./ContactLink";

describe("ContactLink component", () => {
  const props = {
    contactEmail: "housing.rents@hackney.gov.uk",
    contactNumber: "020 8356 3100 (Mon-Fri 9am - 5pm)",
  };
  it("should render properly", () => {
    const { asFragment } = render(<ContactLink {...props} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
