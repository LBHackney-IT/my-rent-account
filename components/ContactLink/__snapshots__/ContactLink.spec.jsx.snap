// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactLink component should render properly 1`] = `
<DocumentFragment>
  <div>
    <p
      class="govuk-body"
    >
      If you have an account query, please email us at 
      <a
        href="housing.rents@hackney.gov.uk"
      >
        housing.rents@hackney.gov.uk
      </a>
      , or call us on 020 8356 3100 (Mon-Fri 9am - 5pm)
    </p>
  </div>
</DocumentFragment>
`;
